### ISSUE TITLE
Less verbosity

### ISSUE THREAD

**Issue by kberry** [ho-tex/hyperref#9]
(reposted from https://puszcza.gnu.org.ua/bugs/index.php?190, author    Cristian Klein <cklein>)

TL;DR

Provide an option "trustdriverautodetect" to disable the autodetected driver message and have a cleaner console output.
Long version

Some users prefer having the LaTeX console as clean as possible, to allow them to focus only on important messages, such as warnings and errors. Hyperref currently displays a message of the form "Package hyperref Message: Driver (autodetected): hpdftex". For some users, this message is important only once, when setting up a new LaTeX project, therefore, they may consider it unimportant and would prefer disabling it.

To reconcile the two categories of users, those who want to see this output and those who do not, I propose adding a new option to hyperref, called "trustdriverautodetect". If the users does not pass this option, hyperref defaults to showing the driver autodetect message. Then, when users are confident that this is no longer needed, they can pass "trustdriverautodetect=true", to instruct hyperref not to show the autodetect message any more.

The attached patch implements this proposition. 

[hyperref.diff.txt](https://github.com/ho-tex/hyperref/files/271120/hyperref.diff.txt)

**Comment by davidcarlisle** [comment]
I dpn't think this is worth yet another option and the extra documentation that needs, I just unconditionally sent it to the log not the terminal.
